config/filters.js

```diff
@@ -1,145 +1,10 @@
 module.exports = [
-	'.wmv',
-	'.wma',
-	'.z',
-	'childporn',
-	'cjhildporn',
-	'child porn',
-	'pedo',
-	'1yo',
-	'2yo',
-	'3yo',
-	'4yo',
-	'5yo',
-	'6yo',
-	'7yo',
-	'8yo',
-	'9yo',
-	'10yo',
-	'11yo',
-	'12yo',
-	'13yo',
-	'14yo',
-	'15yo',
-	'16yo',
-	'17yo',
-	'jailbait',
-	'tetitas',
-	'masha',
-	'marsha',
-	'mafiasex',
-	'pedoman',
-	'pedowoman',
-	'kidcam',
-	'hebefilia',
-	'bebefilia',
-	'videoangels',
-	'hussyfan',
-	'kiddyfuck',
-	'hardcandy',
-	'pedo',
-	'childlover',
-	'ped0',
-	'pedofilia',
-	'pedofila',
-	'petite',
-	'rape',
-	'preteen',
-	'preteens',
-	'preteenz',
-	'lolita',
-	'lolli',
-	'mashaworld',
-	'lolitas',
-	'eurololita',
-	'eurololitas',
-	'rape',
-	'Nymphets',
-	'Nephrene',
-	'babyshivid',
-	'pedofilo',
-	'kiddy',
-	'hairless',
-	'incest',
-	'incesto',
-	'lolifuck',
-	'lolimania',
-	'infant',
-	'diaper',
-	'pedomother',
-	'pedodad',
-	'pedomom',
-	'pedofather',
-	'pedosister',
-	'pedobrother',
-	'daddaughter',
-	'pedoland',
-	'ninas',
-	'nenas',
-	'bebitas',
-	'pornokid',
-	'pornokind',
-	'kinderporn',
-	'kinderfiker',
-	'kinderficker',
-	'littlegirl',
-	'littleboy',
-	'kid sex',
-	'kids sex',
-	'underage',
-	'lolitabay',
-	'pornololita',
-	'pornololitas',
-	'1y',
-	'2y',
-	'3y',
-	'4y',
-	'5y',
-	'6y ',
-	'7y',
-	'8y',
-	'9y',
-	'10y',
-	'11y',
-	'12y',
-	'13y',
-	'14y',
-	'15y',
-	'16y',
-	'17y',
-	'nablot',
-	'1años',
-	'2años',
-	'3años',
-	'4años',
-	'5años',
-	'6años',
-	'7años',
-	'8años',
-	'9años',
-	'10años',
-	'11años',
-	'12años',
-	'13años',
-	'14años',
-	'15años',
-	'16años',
-	'17años',
-	'1yrs',
-	'2yrs',
-	'3yrs',
-	'4yrs',
-	'5yrs',
-	'6yrs',
-	'7yrs',
-	'8yrs',
-	'9yrs',
-	'10yrs',
-	'11yrs',
-	'12yrs',
-	'13yrs',
-	'14yrs',
-	'15yrs',
-	'16yrs',
-	'17yrs',
-];
+'torrentday',
```

Comment:
This is something that I don't think we want to include since this would limit the torrents that went into elasticsearch.

Reply:
didnt realise that was in there:/